### ISSUE TITLE
adopt a cooperative operating agreement

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#72]
It's time. Gittip right now is an LLC that is wholly owned by me. Gittip is _de facto_ owned by the subset of our users that think of themselves as owning Gittip. We need to make this real _de jure_. I think we should be able to do this by writing ~~bylaws~~ _an operating agreement_ for the current LLC.

**Comment by whit537** [comment]
>Scott Bader was founded in 1921 and conventionally managed for 30 years.  However Ernest Bader, the founder, was never comfortable with a capitalist governance structure. Having become Quakers, Ernest and his family believed that ethical and moral action to improve this world was vital. For many years they sought a 'third way' of distributing the capital of the company in a less divisive manner and a method by which all employees could have a 'real' say in the company. They believed that labour should employ capital, acknowledging the equality of everyone as individuals and that only a restructure of the way industry was managed would bring this about.
>
In 1951 a radical decision was made resulting in the creation of The Scott Bader Commonwealth, which was founded on Quaker Principles. 

http://www.scottbader.com/about-us/11/our-history

> The people that work at Scott Bader can become Members of The Commonwealth and as such become trustees-in-common of the company assets. Each generation must ensure the ongoing success of the organisation so that future generations can benefit from its continued existence.

http://www.scottbader.com/about-us/45/our-governance-structure

([h/t](https://en.wikipedia.org/wiki/E._F._Schumacher#Thinking_outside_the_box))

**Comment by whit537** [comment]
https://github.com/colab-coop/bylaws/

**Comment by whit537** [comment]
http://www.co-oplaw.org/topics-2/cooperative-bylaws/

**Comment by whit537** [comment]
Alright, let's start pushing on this a little bit.

> [H]ere are the bylaws of our random consumer/shared-services co-op: http://member.datacommons.coop/bylaws. 

Thanks, @paulfitz! :-)

> You end up with member-owners who use the co-op's services and elect a board that directs the co-op, and (optionally) patrons who use the services of the co-op without being members.

Yes, that's the general direction we want to go. 

> There are professional co-op developers out there who are very helpful at organizing co-op conversions (disclaimer: I'm married to one of them).

We may in fact be interested in talking to the person you're married to. :-) Could you provide contact info here, or privately to chad@zetaweb.com?

I also have a card in front of me for a Pittsburgh lawyer with expertise in worker-owned co-ops, planning to call next week.

> I wonder also if people like AORTA http://www.aorta.coop/ might also be able to help with the overall situation.

Heh, I recognize Esteban Kelley from the Platform Cooperatives conference last week (#384). :-)

**Comment by whit537** [comment]
> The nutshell at the tip of the iceberg is that as a PA LLC, the Operating Agreement is the definition of the entity. Changing it (by whatever means are currently defined in the agreement) would change the rules of how Gittip is defined.

@phoemke Right, okay, so ... bylaws for a corporation, and an operating agreement for an LLC, ya? Gratipay, LLC was formed in 2002, and if we ever had an operating agreement, it's now lost to the sands of time. Let's keep this as a question for the lawyer (once we hire one), what dance we'll need to do to adopt a new operating agreement, given our current situation.

**Comment by whit537** [comment]
I believe next steps here are to start with Data Commons' bylaws, which seem as good as any, and start drafting our own operating agreement. I think we should produce a draft ourselves, and then take it to a lawyer for additional revision. That should save us some money.

**Comment by whit537** [comment]
We want, eventually, millions of members. Right? Is the upcoming Gratipay retreat (#393) going to be our constitutional convention? Our first annual meeting, where we ratify the new operating agreement? O.O

**Comment by whit537** [comment]
I have to say, I really love the Amish way of choosing leadership: they vote on a shortlist of four or so, and then draw straws. I don't know if it's actually drawing straws, but it's random, is the point. I would love to incorporate that into Gratipay's operating agreement. :)

**Comment by whit537** [comment]
"[How to make an air lottery ball machine](https://www.youtube.com/watch?v=iIB_98W2kFg)"—from the House of Hacks, where they're "Living la vida Linux"! :dancer: :penguin: 

[![screen shot 2015-11-20 at 10 10 20 pm](https://cloud.githubusercontent.com/assets/134455/11316404/b4e16afa-8fd3-11e5-9b5c-5289552c08da.png)](https://www.youtube.com/watch?v=iIB_98W2kFg)

**Comment by whit537** [comment]
I would also love to include a reference to [XKCD 221](https://xkcd.com/221/). :-)

[![random_number](https://cloud.githubusercontent.com/assets/134455/11316428/a74a473a-8fd4-11e5-8666-a10671e63586.png)](https://xkcd.com/221/)

**Comment by paulfitz** [comment]
If you do want to check in with some co-op developers, Keystone covers Pennsylvania http://www.kdc.coop/ - I can recommend Jim Johnson there, he was instrumental to the Data Commons getting off the ground. My wife is Noemi Giszpenc of the Cooperative Development Institute http://www.cdi.coop (she presented about the Data Commons at the Platform Coop conf btw).

Obviously I'm biased, but I'm a fan of co-op devs. They have experience of how different ways of organizing a co-op can play out. Gratipay is unique but I bet there is still relevant precedent to draw on and avoid.

**Comment by whit537** [comment]
> My wife is Noemi Giszpenc of the Cooperative Development Institute http://www.cdi.coop (she presented about the Data Commons at the Platform Coop conf btw).

Ah, nice! Looks like she was part of the "Actual Platform Co-ops" session, which unfortunately I skipped (in order to pursue a serendipitous conversation about #319).

> Obviously I'm biased, but I'm a fan of co-op devs.

Instead of lawyers, you mean?

> If you do want to check in with some co-op developers, Keystone covers Pennsylvania http://www.kdc.coop/ - I can recommend Jim Johnson there, he was instrumental to the Data Commons 

Cool, thanks. We may have worked with KDC when I helped start PLOW, a growers' cooperative. That was 10+ years ago, though, so the details are getting fuzzy. :-)

**Comment by whit537** [comment]
![screen shot 2015-11-21 at 11 35 41 am](https://cloud.githubusercontent.com/assets/134455/11319415/164acc30-9044-11e5-92fe-594c5729ac36.png)
![screen shot 2015-11-21 at 11 35 52 am](https://cloud.githubusercontent.com/assets/134455/11319416/164ab146-9044-11e5-9836-cd4ba997dfb0.png)
![screen shot 2015-11-21 at 11 35 55 am](https://cloud.githubusercontent.com/assets/134455/11319414/164a4904-9044-11e5-8eaa-2c6a5776c01c.png)

**Comment by paulfitz** [comment]
> > Obviously I'm biased, but I'm a fan of co-op devs.

> Instead of lawyers, you mean?

Not instead of, but in addition to. For the Data Commons, our lawyer was very helpful in encoding our ideas of how ownership would work, and the co-op developers were key in coming up with and debugging those ideas in the first place.

**Comment by whit537** [comment]
Cool. We never would've made it through the Gratipocalypse without a good lawyer. They have their place.

**Comment by whit537** [comment]
Looking into Enspiral further, especially after #403.

 - http://p2pfoundation.net/Open_Cooperatives#The_Enspiral_Network
 - "[Inventing a New Organizational Operating System](https://www.youtube.com/watch?v=4ABqCgKi4X0)"
 - "[No Boss Does Not Mean No Leadership](https://medium.com/enspiral-tales/no-boss-does-not-mean-no-leadership-c4c97c660252)"
 - "[How to Grow Distributed Leadership](https://medium.com/enspiral-tales/how-to-grow-distributed-leadership-7f6b25f0361c)"

**Comment by whit537** [comment]
One of the questions I have is how a million-person co-op that operates primarily on the Internet has an annual meeting, with election of officers. Is the traditional board structure relevant for what we're trying to accomplish?

**Comment by whit537** [comment]
> Contrary to what we would like to believe, there is no such thing as a structureless group. Any group of people of whatever nature that comes together for any length of time for any purpose will inevitably structure itself in some fashion. The structure may be flexible; it may vary over time; it may evenly or unevenly distribute tasks, power and resources over the members of the group. But it will be formed regardless of the abilities, personalities, or intentions of the people involved. [...]
>
This means that to strive for a structureless group is as useful, and as deceptive, as to aim at an "objective" news story, "value-free" social science, or a "free" economy. A "laissez faire" group is about as realistic as a "laissez faire" society; the idea becomes a smokescreen for the strong or the lucky to establish unquestioned hegemony over others. This hegemony can be so easily established because the idea of "structurelessness" does not prevent the formation of informal structures, only formal ones.

http://www.jofreeman.com/joreen/tyranny.htm

**Comment by whit537** [comment]
What are the operating agreements of the various [Enspiral Ventures](http://www.enspiral.com/ventures)? What can we borrow from them? This would obviously inform #421 as well.

**Comment by whit537** [comment]
Copying over from https://github.com/gratipay/inside.gratipay.com/issues/242#issuecomment-161046259 ...

>To: Eric
>Subject: help with international co-op?
>
>> Chad Whitacre here from Gratipay, just left you a voicemail as well. We met after Jessica Jackley's Speak Freely a little while back. My startup needs help a) organizing as a cooperative and b) figuring out how to pay ourselves. This is within your wheelhouse, yeah? Do you have time in the next couple weeks for a conversation?
>
Will include #411 in my next communication.

**Comment by whit537** [comment]
To: Janelle Orsi
Subject: lawyer recommendations?

> Chad Whitacre here from Gratipay. Great to meet you in person at Platform Cooperatives! :-)
> 
I'm writing to ask if you have any recommendations for a lawyer to help Gratipay write a cooperative operating agreement. We need something that works internationally, and makes joining and leaving the cooperative really easy. We also need to figure out the best way to structure payments to our owner-members. Here's our GitHub ticket:
> 
https://github.com/gratipay/inside.gratipay.com/issues/72
> 
Whatever we figure out for ourselves, we'd like to be able to offer as a product to our users, as well, which is what this ticket is about:
> 
https://github.com/gratipay/inside.gratipay.com/issues/242
> 
P.S. I'll need to at least summarize our conversation publicly with the Gratipay community. If you'd rather answer directly on GitHub, feel free! :-)
> 
Thanks, Janelle! :-)

**Comment by JanelleOrsi** [comment]
Since I'm not caught up with this thread yet (apart from how to make a lottery ball), quick question: are you a Pennsylvania LLC? 

If so, you'll either need a PA lawyer, or someone to team up with a PA lawyer to make sure everything is compliant with the PA LLC statute (I'm guessing PA law says that out-of-state lawyers can't practice there unless they are working with a PA lawyer on a client matter). 

Possibilities in or outside of PA: 
* [Praveen Kosuri](https://www.law.upenn.edu/cf/faculty/pkosuri/) runs a law clinic at Penn.
* For some reason, I thought that either Parag Khandhar or Dorcas Gilmore from [this firm](https://twitter.com/solidaritylaw) is certified in PA, but maybe it's just in Maryland. In any case, they could be good. 
* [Jason Wiener](http://www.jrwiener.com/about-jason/) - coop lawyer certified in Colorado and NY - he represents Members Media, which was present at the Platform Coop conference
* [Renee Lloyd](https://www.linkedin.com/in/reneelloyd), who you will recognize from the Platform Coop conference
* Any of the 18 lawyers who are [Fellows at the Sustainable Economies Law Center](http://www.theselc.org/fellows), since they also have an open door to ask SELC staff attorneys for mentorship on things like this
* I'm happy to chip in an hour of brainstorming if you'd like, since you'd already paid my fee in the form of a penny with a heart in it.

**Comment by whit537** [comment]
Thanks for all the recommendations, @JanelleOrsi! We are a Pennsylvania LLC, yes. I look forward to pursuing these leads and seeing where they take us! :-)

**Comment by whit537** [comment]
> I'm happy to chip in an hour of brainstorming if you'd like, since you'd already paid my fee in the form of a penny with a heart in it.

@JanelleOrsi Hah! :D Thank you! Any of these times work for you?

Dec 3 (tomorrow) at 11am PST 
Dec 9 at 11am
Dec 17 at 11am

If we make it a Hangout on Air then others can follow along. Also happy to Skype privately and summarize here.

**Comment by webmaven** [comment]
I'd be quite interested in following along with a Hangout (or watching a recording, If I can't attend synchronously).

**Comment by JanelleOrsi** [comment]
None of those times work, but the 11th, 15th (morning), and 18th work. Hangout on Air is fine with me.

**Comment by whit537** [comment]
Let's go for the 11th, @JanelleOrsi. 11am PST okay?

**Comment by whit537** [comment]
In other news: I got a voicemail back from Eric, hoping to follow up tomorrow.

**Comment by JanelleOrsi** [comment]
11am PST on the 11th is good. Just let me the Hangout link.

BTW, any support I give is in my personal capacity, not on behalf of the Sustainable Economies Law Center. Just an important distinction I need to make.

**Comment by whit537** [comment]
> BTW, any support I give is in my personal capacity, not on behalf of the Sustainable Economies Law Center. Just an important distinction I need to make.

Understood. :-)

> 11am PST on the 11th is good. Just [email] me the Hangout link.

Cool, will do! Thank you! :-)

P.S. Feel free to unsubscribe from this ticket if it's distracting for you. I anticipate that it's going to see a fair amount of traffic. Also feel free to keep listening/participating! :-)

**Comment by whit537** [comment]
Phone tag with Eric.

**Comment by whit537** [comment]
I have a call set up with Eric for 9am EST on Thursday.

**Comment by whit537** [comment]
I did a little reading on the differences between [a cooperative and an employee-owned company](http://www.cdi.coop/coop-cathy-worker-coops-esops-difference/) (we want a cooperative), and also between [cooperatives and collectives](http://cultivate.coop/wiki/Differences_Between_Worker_Cooperatives_and_Collectives) (they're orthogonal: cooperatives are about ownership, collectives are about management).

**Comment by whit537** [comment]
Skimming through @JanelleOrsi's [recommendations](https://github.com/gratipay/inside.gratipay.com/issues/72#issuecomment-161341705), I am seeing: New Jersey, New York, Massachusetts, California, North Carolina, Washington, New Hampshire, Michigan, Illinois, and Colorado. It's possible that some of these folks are also admitted in PA. Let's dig deeper if other leads don't pan out.

I found a number for Gilmore Khandhar on [their Facebook](https://www.facebook.com/solidaritylaw/). I called and left a voicemail.

To: Praveen Kosuri
Subject: converting to a cooperative?

> Praveen,
>
Greetings! My name is Chad Whitacre, and I run a payments start-up called Gratipay.
>
I'm looking for counsel converting Gratipay, which is a single-member Pennsylvania LLC, into a cooperative. Janelle Orsi from the Sustainable Economies Law Center gave me your name. Is this something you might be able to advise us on? If so, could we set up a time in the next few weeks for an initial phone conversation? If not, would you have any recommendations for attorneys we might contact?
>
Thanks!
>
P.S. I'll need to at least summarize our email exchange publicly for the Gratipay community; here's the link to [our public internal conversation](https://github.com/gratipay/inside.gratipay.com/issues/72). Let me know if you'd prefer that I copy your responses verbatim to avoid misrepresentation.

**Comment by whit537** [comment]
Just got off the horn with Eric. I've got a page of notes, and I hope to circle back here to summarize sometime after #399.

**Comment by whit537** [comment]
I've heard back from Jim Johnson (KDC) in email. We're arranging a phone call in the next couple weeks. The #411:

> P.S. I'll need to at least summarize our email and phone conversations publicly for the Gratipay community. Here's our [internal public discussion thread](https://github.com/gratipay/inside.gratipay.com/issues/72) for your reference. If you'd prefer that I copy your messages verbatim, let me know! :-)

Call w/ @JanelleOrsi in 12 minutes ... :-)

**Comment by whit537** [comment]
Call w/ @JanelleOrsi: https://www.youtube.com/watch?v=vnS4Hf9Ro3Q.

**Comment by whit537** [comment]
Alright! Thanks for the call, @JanelleOrsi! :dancer: Lots to think about, both for this ticket and for #404, etc. and for general "How do we do this thoughtfully?" type questions. Looking forward to a cartoon and the owner/employee shtuff! :-)

**Comment by whit537** [comment]
Received in private email; shared with permission:

>Cartoon about governance:
https://www.youtube.com/watch?v=kJc60wCbL70&list=PL5Qc8ueoLShsagZIDjThmjSabFm7pXQxq&index=10
>
A couple things I've written about employment law:
http://communityenterpriselaw.org/employment/
http://www.co-oplaw.org/topics-2/employment-law/

**Comment by whit537** [comment]
I have a call scheduled with Jim Johnson (KDC) at 2pm EST on December 22.

**Comment by whit537** [comment]
Heard back from Praveen:

> We might be able to assist you. We are a free transactional legal clinic staffed by upper-level law students.  As such, we take on new clients at the beginning of each semester.

We're scheduling a (closed) call for early January.

**Comment by whit537** [comment]
> Cartoon about governance:

I finished watching [this](https://www.youtube.com/watch?v=kJc60wCbL70) on the bus this morning. Lots to think about! Is governance life, or does governance *enable* life? Hmm .... Lots of good practical advice, as well, about [conflicting goals](https://www.youtube.com/watch?v=kJc60wCbL70#t=18m08s), the [limits of consensus by unanimity](https://www.youtube.com/watch?v=kJc60wCbL70#t=24m42s), and the possibility of [loose organizational structures](https://www.youtube.com/watch?v=kJc60wCbL70#t=26m05s), e.g.

I'm following it up with [*The Mondragon Experiment*](https://www.youtube.com/watch?v=2zMvktpKDmo). Mondragon is often cited as an example of cooperative success, and I've never engaged with them very deeply. Seems appropriate to do so at this time.

**Comment by whit537** [comment]
For reference, here's what @Changaco came up with for Liberapay's [Bylaws](https://github.com/liberapay/liberapay.org/blob/da4e93646aac9792191e8847be414b16659a2a58/en/bylaws.md) and [Rules of procedure](https://github.com/liberapay/liberapay.org/blob/master/en/rules_of_procedure.md).

**Comment by JanelleOrsi** [comment]
Here's [a sequel](https://www.youtube.com/watch?v=tEVc1QSKBGw&feature=youtu.be) to that Governance is Life video. Chad, this is similar to what I tried to send via dropbox.

**Comment by Changaco** [comment]
> For reference, here's what @Changaco came up with for Liberapay's [Bylaws](https://github.com/liberapay/liberapay.org/blob/da4e93646aac9792191e8847be414b16659a2a58/en/bylaws.md)

A quick note: they're based on French law, and they're for a non-profit organization, not for a cooperative or an LLC.

**Comment by whit537** [comment]
Thanks, @Changaco. Noted. :-)

And thanks for the video, @JanelleOrsi. Even more food for thought. The organization you [mention](https://www.youtube.com/watch?v=tEVc1QSKBGw#t=12m28s) in connection with intrinsic motivation and caps on extraction sounds like a kindred spirit to Gratipay! :-) In fact, the ["take-what-you-want" compensation](https://gratipay.com/about/features/payroll) that we're trying so hard to get back to was indeed designed to "rejuvenate" intrinsic motivation, and to use the social pressure of fairness within a strongly cohesive group precisely as a cap on extraction.

One thing I'm chewing on is the way in which formal provisions that work well in a situation where everyone is a full-time employee (or a board member roughly equivalent to the others) seem not to fit the FOSS-ish situation we've got here, where not only compensation (output) but also effort (input) is voluntary, resulting in a fat head and a long tail of inputs. How do we prescriptively match outputs to inputs when inputs are voluntary? What does it mean to say "everyone gets equal pay" when the range of effort varies so widely? Instead of a definitive formal approach to legally *enforce* the outcomes we want, I think we end up needing to rely on designing systems and "[cultural technologies](http://blog.loomio.org/category/cultural-technology/)" to *strongly encourage* the behavior we're looking for. This goes for other aspects as well, such as "restorative" efforts.

**Comment by whit537** [comment]
As mentioned at https://github.com/gratipay/inside.gratipay.com/issues/242#issuecomment-166604631, I'm now wondering whether full-fledged legal ownership is in fact the best way to encode Gratipay's openness. Legal ownership is rigid and inflexible. Don't we want to maintain our FOSS-style fluidity?

**Comment by whit537** [comment]
FOSS projects have "core committers" as well as non-profits with boards, with a good deal and possibly the bulk of the activity happening outside (or within?) the more formal structure.

**Comment by whit537** [comment]
How about this for an outline:

1. A minimalist six-member **board** formally owns Gratipay. The members of the board are trustees of Gratipay on behalf of the Gratipay community. The board's responsibilities include:

 - Ensuring the continuity of Gratipay as a legal entity.
 - Ensuring the continued forkability of Gratipay.
 - Overseeing the selection of the CEO.

1. One-third of the board is replaced every other year, as follows:

 - The ~users of Gratipay elect one replacement from among themselves.
 - The four continuing board members appoint one replacement.

1. Gratipay has a rotating **CEO** that acts as [benevolent dictator](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life)—but not for life. The CEO rotates every two years (out of phase with board selections), and is therefore a "benevolent dictator for now" (BDFN).

1. The CEO is selected at random from among three candidates:

  - The board appoints one candidate.
  - The current CEO may run again, or may appoint one candidate.
  - The ~users of Gratipay elect one candidate from among themselves, or two if the CEO was fired or resigned or didn't appoint a candidate.

1. The board grants the CEO broad control to run the company. Accountability comes from the Gratipay community as a whole (which, of course, includes the board) via our public collaboration channels. Forking is the answer for irreconcilable disagreements.

**Comment by whit537** [comment]
And sorry @dmk246 for the accidental assignment! :O

**Comment by whit537** [comment]
Okay! Just wrapped a call with Jim Johnson from KDC. Here are the [call notes](https://docs.google.com/document/d/1noEf1Te7B2N_9giF_Eca-6tCVZQ8tv77CoSjcKd9fME/edit?usp=sharing), which includes snippets from this ticket along with bullet points on organizational forms to consider. Perhaps the most fun off-the-wall idea: we could become a [501(d) Apostolic Association](https://www.irs.gov/irm/part4/irm_04-076-029.html)! :dancer: In general, Jim said that we're "already very advanced" in terms of our analysis of the playing field and our position. We talked about some of the challenges of multi-stakeholder cooperatives (namely, balancing competing interests), along with ways to enfranchise stakeholders without necessarily giving them a board seat. For example, we could combine a 501(c)3 that represents community interests, with a separate cooperative owned by the staff. Some worker co-ops also allow for no-owner workers to handle people who work less than full time. If it comes to it, Jim mentioned the possibility of approaching a state corporation office, floating whatever we come up with, and seeking a "no action letter," meaning they wouldn't expect to take any action against us for doing what we proposed.

In terms of services available, KDC can help groups get "lawyer-ready" and "accountant-ready." They are available to help us workshop our bylaws/operating agreement and also to look over our [finances](https://github.com/gratipay/inside.gratipay.com/labels/Accounting). They work hourly at $75/hr. Jim is up for using Hangouts for future work together where it makes sense.

**Comment by whit537** [comment]
Over on https://github.com/gratipay/inside.gratipay.com/issues/432 we're having a conversation about decoupling payments and payroll on Gratipay, which has the implication that we would probably loosen up our requirements on who can be on Gratipay. That has implications for this ticket, because one goal here is to ensure the coherence of Gratipay's brand identity over time. Per [Ostrom's Principles](https://en.wikipedia.org/wiki/Elinor_Ostrom#Design_principles_for_Common_Pool_Resource_.28CPR.29_institutions), proper commons design begins with clear boundaries. If under #432 we introduce tiers of ~users on Gratipay, how would that affect the design of our structure here? I want to prevent—well, discourage, at least—identity drift. Perhaps that becomes an explicit responsibility of the board? Perhaps the ~users entitled to vote are those participating (owning? giving to? taking from?) "Gratiplatinum" organizations?

**Comment by whit537** [comment]
Gratipay, LLC does not have a written operating agreement—and is not required to by PA law, afaict. Here's [the law we're organized under](http://www.legis.state.pa.us/cfdocs/legis/LI/consCheck.cfm?txtType=HTM&ttl=15&div=0&chpt=89), and here's what it has to say about written operating agreements:

> The operating agreement of a limited liability company need not be in writing except where this chapter refers to a written provision of the operating agreement.

**Comment by whit537** [comment]
So it's fine that we haven't had a written operating agreement. This ticket is about adopting a written operating agreement.

**Comment by whit537** [comment]
Are we proceeding with the LLC we've already got? Or are we going to move to Estonia (#438) in the process?

**Comment by whit537** [comment]
Working with the existing LLC would be more convenient in the short run, because we already have it and I'm more familiar with PA LLCs. However, an Estonian OÜ would seem on the face of it to be a better long-term fit for the international character of Gratipay.

**Comment by whit537** [comment]
Karl Fogel's chapter on "[Social and Political Infrastructure](https://books.google.com/books?id=0vbr7xvvzjgC&pg=PA87)" has much to contribute to this ticket.

**Comment by whit537** [comment]
Here's another helpful post on [forkability](http://a3nm.net/blog/crowdsourcing_ethics.html). I think forkability is essential to the structure we want to put in place.

**Comment by webmaven** [comment]
@whit537:
> How about this for an outline:
> 
> 1. A minimalist six-member **board** formally owns Gratipay.

Hmm. What is your current thinking on Gratipay becoming a worker-owned coop, and making @rohitpaulk a worker/owner with one share? Has that gone by the wayside?

Also:
> - The ~users of Gratipay nominate one candidate from among themselves by election, or two if the CEO was fired or resigned or didn't nominate a candidate.

Who has the power to fire the CEO?

> Karl Fogel's chapter on "Social and Political Infrastructure" has much to contribute to this ticket.

Absolutely, but you might prefer the latest version: http://producingoss.com/en/social-infrastructure.html

**Comment by whit537** [comment]
> Who has the power to fire the CEO?

When I wrote that, I was thinking the board did, but now I'm thinking the answer is to fork rather than fire.

**Comment by whit537** [comment]
> Hmm. What is your current thinking on Gratipay becoming a worker-owned coop, and making @rohitpaulk a worker/owner with one share? Has that gone by the wayside?

Yes. As has surfaced over at https://github.com/gratipay/inside.gratipay.com/issues/242#issuecomment-171401916, the analysis above at https://github.com/gratipay/inside.gratipay.com/issues/72#issuecomment-166605327, that formal legal ownership is not flexible enough for our FOSS-style fluidity, seems borne out by Loomio's example, where some members are not staff, and some staff are not members. Better to keep the legal structure as slim as possible to avoid such absurdities. As I suggest at https://github.com/gratipay/inside.gratipay.com/issues/242#issuecomment-166650524, "the real sharing of control in Gratipay happens daily here on GitHub, not in an annual meeting."

**Comment by whit537** [comment]
Note that this doesn't mean that we're "structureless," but rather that our structure is implemented at a higher level of abstraction than the legal system.

**Comment by whit537** [comment]
P.S. I've reviewed:

https://en.wikipedia.org/wiki/Chief_executive_officer
https://en.wikipedia.org/wiki/CEO_succession

**Comment by techtonik** [comment]
It is all an awesome reading. I wish I had more time to read it. Would appreciate some writeup. =)

**Comment by whit537** [comment]
Hot on the heels of Holacracy (#404), here's a doc out of Loomio/Enspiral (#420/#421) about "[Agreement-Based Organization](https://docs.google.com/document/d/1c_xWEIay-2jyJ3Rqb6OgTxoZBJfjNW4d6w6ukXyeJk4/edit)," which also references [Sociocracy 3.0](http://sociocracy30.org/the-details/). More thinking to digest towards our own operating agreement.

**Comment by whit537** [comment]
I'd also like to have [PEP](https://www.python.org/dev/peps/pep-0001/) on the table.

**Comment by whit537** [comment]
> Hmm. What is your current thinking on Gratipay becoming a worker-owned coop, and making @rohitpaulk a worker/owner with one share?

My current line of thought is that Gratipay might work best as a [consumer cooperative](https://en.wikipedia.org/wiki/Cooperative#Consumers.27_cooperative):

> A consumers' cooperative is a business owned by its customers. Employees can also generally become members. Members vote on major decisions and elect the board of directors from among their own number.

"Consumer" isn't quite the right word, though (and certainly not "retail"), because our [primary customer](http://inside.gratipay.com/big-picture/customers/) is the Team. In other words, we would be cooperatively owned by the organizations that use Gratipay to collect and distribute money.

**Comment by whit537** [comment]
@nathanairplane Would you be up for an open call to discuss converting Gratipay into a customer-owned cooperative? You are [our customer](https://gratipay.com/internet-of-ownership/) with perhaps the clearest interest in this topic. ;-)

I [understand](https://twitter.com/dasharp/status/740129524436733952) you're traveling at the moment ... looking a little further out, do any of these times work for you?

 - Tuesday, June 21 at 9:30 AM Boulder time (MDT)
 - Tuesday, June 28 at 9:30
 - Friday, July 1 at 9:30

[Also sending in private email since I'm not sure you watch GitHub notifications ...]

**Comment by nathanairplane** [comment]
Thanks, @whit537. I'd be happy to help if I can, though I'm a reporter more than a business consultant, so I guess I'd be doing this more in my role as a customer. Of these, the only one that works is 6/28 at 9:30 Mountain Time. Shall we plan on that?

**Comment by whit537** [comment]
> I'd be happy to help

Yessssss! 💃 

> I'm a reporter more than a business consultant [...]

... though a reporter [of questionable objectivity](https://en.wikipedia.org/wiki/Nathan_Schneider#Coverage_of_Occupy_Wall_Street). ;-)

> [...] so I guess I'd be doing this more in my role as a customer.

Indeed, and perhaps even a customer-_owner_ by the end of the exercise! :-)

> Of these, the only one that works is 6/28 at 9:30 Mountain Time. Shall we plan on that?

Yes! Are you okay with using Google Hangouts on Air [again](https://www.youtube.com/watch?v=YgxkGELtKME)? I'm up for a FLOSS alternative, if you know of one that likewise provides streaming and archiving.

In terms of an agenda, my thinking is that I will draft a proposal for how Gratipay could be converted to a cooperative, and then you and I can dig into that on the call to see if it really holds water as a "platform cooperative." I want to see what emerges when we try to combine your vision for what a platform cooperative should be, and my vision for what an excellent product should be.

**Comment by nathanairplane** [comment]
Sounds good. Goog would be fine. I usually prefer to use Jitsi @ meet.mayfirst.org, of which I'm a member, but I don't think it allows for archiving unfortunately.

**Comment by whit537** [comment]
Okay. I'll send you a join link in private email on 6/28 at 9:30 Mountain Time. Huzzah! 💃

**Comment by whit537** [comment]
Bumping a few hours to account for jet lag. :-)

**Comment by whit537** [comment]
https://www.youtube.com/watch?v=tT452Sy3MI0

**Comment by whit537** [comment]
(Sorry, neglected to post that before starting!)

**Comment by nathanairplane** [comment]
Thanks for the invigorating conversation! I'm really excited by the potential, and I think Gratipay could become a powerful force for jumpstarting an [open co-op](http://commonstransition.org/open-co-ops-inspiration-legal-structures-and-tools/), platform co-op ecosystem.

As I said, I think Enspiral could be a really important model in some respects—particularly in their model for co-budgeting and for their stakeholder arrangement of individual "members" and "contributors" plus affiliated "ventures." I've just started a little archive of Enspiral stuff [here](http://nathanschneider.info/wiki/commons/enspiral); their resources on GitHub might be particularly useful for you.

**Comment by whit537** [comment]
Invigorating, indeed! I also ended up having an appointment at the top of the hour (I had tried to bump it but failed), so don't feel bad that we had to cut our time short. :)

I appreciate [framing](https://www.youtube.com/watch?v=tT452Sy3MI0#t=9m25s) the question as one of "ownership design." I'm interested to see what's said in [_Owning Our Future_](http://www.marjoriekelly.com/books/owning-our-future/); what this framing leads me towards is the question, _why_? What is it that Gratipay's legal structure needs to accomplish?

One of my take-aways from the call is that there are three reasons why we need to work on Gratipay's legal structure: capital, accountability, and continuity. The most pressing is continuity—ensuring that Gratipay is a stable organization that can survive without me—because we do have at least some answers for capital (namely, revenue :) and accountability (forking). As we design Gratipay's ownership, let's keep these in mind.

Regarding Enspiral, they did come up [above](https://github.com/gratipay/inside.gratipay.com/issues/72#issuecomment-159727340) in this thread (just after Platform Cooperativism, it looks like ;), which led to #421. I appreciate the [suggestion](https://www.youtube.com/watch?v=tT452Sy3MI0#t=52m07s) to connect with Alanna (cf. https://github.com/gratipay/inside.gratipay.com/issues/421#issuecomment-159751078) and Joshua (I have to admit that discovering Enspiral to have had a single founder feels like I'm now in on a secret :o). Perhaps I should try to get to [OS//OS](http://www.opensourceopensociety.com/), after all? :)

Thanks for helping stir this pot! :)

**Comment by whit537** [comment]
> **Could there be a company that nobody owns?**
>
> There's this vision of an open company. One example of this, for instance, is Gratipay, what they're trying to work toward, which is a crowdfunding thing for open-source projects, and it kind of puts an economic layer on open-source development. You could imagine, for instance: what if there were a payment layer on GitHub? People got paid for their engagement with the projects, but there was still a kind of ambiguity about ownership. You didn't need to hold shares. Maybe the project is still kind of floating in another world. Who knows what the legal environment is? That's been appealing to a lot of people in this space. I think it's interesting, but we also live in a society where ownership is the law. Ultimately, even a company like that, if there's value flowing through it, somebody's got to be holding that at some point. I think it _is_ important to make sure that question is addressed _before_ we give up ownership.
>
To be honest, I'm kind of a "property is theft" sort of person. I'm not craving more property. I'm more interested in a commons, but I don't want a commons that's just controlled by Google. I want a commons that emerges from a sense of parity and equity among commoners. While these questions of ownership are kind of frustrating, they're also really important.
>
We see examples of this starting to emerge in the blockchain stuff. I think bitcoin is a really telling example of what happens when people try to pretend that ownership and governance can be evaded through neat algorithms. You end up with a situation where a small number of unaccountable large bitcoin miners—nobody really knows who they are—are controlling the system, and the thing is careening into the ground even while somehow the value keeps going up. That, to me, is not a future that I'm thrilled to jump into right now. On the other hand, that technology could be _beautiful_ for developing really, genuinely democratic practices. But we have to start by thinking about how it works in our _relationships_, and how it can create genuine accountability, rather than trying to jump ship on accountability and ownership from the beginning. I think that's tended to be a dangerous strategy.

@ntnsndr at https://www.youtube.com/watch?v=9lkLaV85Cuo#t=31m02s

**Comment by whit537** [comment]
At https://www.youtube.com/watch?v=tT452Sy3MI0#t=48m16s we sort of dismissed the idea of organizing as a non-profit, but we may have been too quick to do so. I worked with a non-profit once that was in fact "fee-for-service" (their words; funded from revenue, in other words). Perhaps being owned by no-one-and-everyone *is* the right choice for Gratipay? That's the route Liberapay took, for reference.

**Comment by JanelleOrsi** [comment]
Just piping up to say this is a great conversation. Chad and Nathan, I just shared with you a draft of an article I'm writing on legal structures for the commons. If anyone else wants to see it, just email me at janelle@theselc.org

Now I'll put in a plug for a nonprofit structure. My thoughts are: 
1. You can be a **nonprofit corporation without tax exemption,** which basically makes you a corporation that no one owns and that can't distribute profits to private parties (though it can of course compensate people for their work). You pay taxes like a regular corporation, but only on net earnings. I imagine you could build up an operating reserve, pay taxes on that, and then budget to operate at cost so that you don't have much net income each year.

2. You could seek a form of tax exemption other than 501(c)(3). For example, you could potentially pass muster as a social welfare organization under **501c4.** You can't get tax deductible donations, but you have the advantage of not paying tax. 

3. Or you can be a **501c8 fraternal benefit society**, which is what Thrivent Financial is - [check it out here](https://en.wikipedia.org/wiki/Thrivent_Financial). This would make Gratipay a mutual, which is pretty darn close in nature to a cooperative, just without dividend payments. 501c8s have some [quirky legal requirements](http://communityenterpriselaw.org/forming-community-enterprise/nonprofit-organizations-in-the-sharing-economy/#Sections_501c8_and_501c10) (I've written about it here), but it's workable. It might actually be fun, because of the laws requirement to create social ties among members. Yup, you gotta make it "brotherly." The IRS gets to decide whether you are brotherly enough.

4. There's also **501c12** -- which has always seemed to me to be the IRS's strange catch-all for services provided to members at-cost, though the IRS is [fussy about what gets this tax exemption](http://communityenterpriselaw.org/forming-community-enterprise/nonprofit-organizations-in-the-sharing-economy/#Section_501c12). Life insurance mutuals are included, so it seems like other kinds of financial services could work. 

5. And unfortunately, even being a nonprofit doesn't prevent the assets of Gratipay from one day in the future becoming swallowed up by the market. Couchsurfing is my cautionary tale, because it was a nonprofit until the founders sold it to themselves for around $600,000. Ugg, every time I think about it, I get irritated at multiple levels. I believe the attorney general fudged up in letting them appraise the company for such a small amount, and the argument that they couldn't stay a nonprofit just because they couldn't get tax exemption has no basis in the law....I could go on about what is wrong with it, but I won't. I'll only say that if Gratipay wants to be owned by nobody, it would be good to empower a few watchdogs to prevent such a transaction from slipping by anyone. You can give other nonprofits the right to intervene to protect the assets, for example.

**Comment by ntnsndr** [comment]
@JanelleOrsi I don't think I got the draft. I wanna see it! n@nathanschneider.info!

Great stuff. I'll leave it to Janelle to advise on legal structures. One interesting experience of Enspiral is that they felt forced not to go nonprofit b/c NZ charity law doesn't treat open-source projects as charitable. But I don't think we have that problem.

**Comment by whit537** [comment]
> You can be a **nonprofit corporation without tax exemption**

I find this option interesting ([cf.](https://twitter.com/whit537/status/748965673070501888) :), because I think "owned by everyone/no-one" is truest to Gratipay's character ... and I'm not sure we need more organizations trying to get out of paying taxes. ;-)

> Couchsurfing is my cautionary tale

Indeed! @bruceadams [made](https://github.com/gratipay/gratipay.com/issues/101#issuecomment-18071226) this general point when we first looked at adopting a non-profit structure. I resonate with his conclusion:

> The primary defense against a change of control for Gittip is in the human leadership of Gittip. Bits of legal language, such as a corporate bylaw or being a nonprofit might help a little, but real defense is in the people.

As I [brought up](https://www.youtube.com/watch?v=tT452Sy3MI0#t=50m32s) with @ntnsndr and I think also with @JanelleOrsi (though I'm not seeing a transcript on our call above at https://github.com/gratipay/inside.gratipay.com/issues/72#issuecomment-164018346, so I'll need to rewatch to find it), the open-source principle of _forkability_ puts some teeth to @bruceadams' suggestion. If Couchsurfing was an open company with open-source code and a vibrant contributor community, then that would have been a check on the founders selling to themselves for cheap, since others in the community would have been empowered to fork. For the record, Gratipay has already had two forks, so this isn't hypothetical. :)

----------------

P.S. I skimmed the draft chapter—legally binding cartoon slideshows sound _amazing_! I'll try to give it a more thorough read on the bus next week ...

P.P.S. I think if you edit your comment and add another line before the first bullet point then the numbering will be fixed and everything will be right with the world. :)

**Comment by whit537** [comment]
> I don't want a commons that's just controlled by Google

Is Mozilla better than Google? What if Gratipay were a cooperative owned by non-profits?

[WikiMedia](https://wikimediafoundation.org/wiki/Home), [Mozilla](https://www.mozilla.org/en-US/foundation/), [Python](https://www.python.org/psf/), [Django](https://www.djangoproject.com/foundation/), [jQuery](https://jquery.org/), [Node.js](https://nodejs.org/en/foundation/), [FSF](http://www.fsf.org/), [OSI](https://opensource.org/), [SFC](https://sfconservancy.org/), [SPI](http://www.spi-inc.org/), [Linux](http://www.linuxfoundation.org/), [Gnome](https://www.gnome.org/foundation/), ...

I believe @JanelleOrsi has implemented non-profit structures where some board seats are reserved for other non-profits: another option.

cf. #637 for an example of how this direction could aid in raising capital

These orgs can also serve as fiscal sponsors for the tiny-just-starting-out projects.

Questions is: would they see value in co-owning a payments service?

**Comment by whit537** [comment]
It's interesting that non-profit was Enspiral's first instinct: that remains my leaning here as well. We've started to apply for grants (#836 #388 #637), with some hand-waving around our corporate structure. If we win any of the grants, then the path of least resistance will be to convert to a non-profit before actually accepting the funds.

**Comment by whit537** [comment]
I'm back on the fence between non-profit and cooperative. I really like the [cooperative principles](http://www.slvrec.com/content/7-cooperative-principles). Especially if these grants don't come through, a cooperative structure with a crowdfunding campaign (#870) to raise capital from initial membership could be the way to go. I'm thinking of this in terms of capital as well as governance, you see (#405).

**Comment by whit537** [comment]
I'm also thinking of this in light of the [open organization conversation](https://github.com/gratipay/inside.gratipay.com/issues/757#issuecomment-256234457). Like, how strongly defined do we want to be.

**Comment by ntnsndr** [comment]
This may come as a surprise, but I don't think you need to be married to the co-op legal structure. Many co-ops don't use it. For instance, my hosting/cloud co-op, May First, is a nonprofit. Members still support it with dues. I think you could practice cooperative principles through a nonprofit legal structure, especially in terms of governance. I think that the open-work principles are vital to this project, and you should prioritize them as much as you can. You're forging the way to the new cooperativism.

**Comment by whit537** [comment]
@nobodxbodon at  https://github.com/gratipay/inside.gratipay.com/issues/836#issuecomment-258776074:

> Sorry to bring this up at this moment. Pitifully I missed the discussion of switching to non-profit, which I vaguely recall but can't find the exact issue (could you point me to it?). I heard from a speech about non-profit business that, non-profit usually is harder to run than for-profit business, which I believe you already know well. IMO, a for-profit business can stand out more with the openness and transparency, compared with a non-profit can (openness in non-profit is basically obligation).
>
Another aspect is, there are way more for-profit businesses than non-profit, and if Gratipay can find a successful business model, it will be more influential, especially if it's potentially reproducible by other for-profit businesses.

Follow-up from @mattbk:

>> if Gratipay can find a successful business model, it will be more influential, especially if it's potentially reproducible by other for-profit businesses.
>
👍 I think we want to be influential and show that this stuff can work (along with letting people run experiments*, of course).
>
I'd argue additionally that, unless there is a pressing reason, there is no need to move toward nonprofit right now.
>
*I don't know who brought this up (maybe @rohitpaulk? My search-fu is weak this morning), but one potential use for allowing multiple split-up-the-money methods (beside take-what-you-want) and multiple ways of giving (bundles, corporate, others we've brainstormed) is to see what works financially and what doesn't. With enough entities using Gratipay, it could be a good resource for some economic papers.

**Comment by whit537** [comment]
Converting to a non-profit reticketed as https://github.com/gratipay/inside.gratipay.com/issues/916.

**Comment by whit537** [comment]
I'm partway through [_Ours to Hack and to Own_](http://www.orbooks.com/catalog/ours-to-hack-and-to-own/), which has a lot to say here. I also want to digest the governance/ownership chapter of [_The Art of Community_](http://www.artofcommunityonline.org/).

**Comment by whit537** [comment]
> All large cooperative and membership organizations (nonprofits, worker-owned enterprises, etc.) that seek to govern themselves democratically face a perennial problem. How can an organization maintain member interest in matters of organizational governance year after year, and avoid management capture?

http://www.coopwatercooler.com/discussions//4m79q1m88fwktufew30pyeg3k4i9zh

[Sortition](https://en.wikipedia.org/wiki/Sortition)! 💃

**Comment by whit537** [comment]
Here is lots of [opinionated theory and some(?) howto on worker cooperatives](https://development.abolishhumanrentals.org/). And here is a documentary about worker cooperatives called, [*Can We Do It Ourselves?*](https://www.youtube.com/watch?v=Cqt61tnCKnU).

**Comment by whit537** [comment]
I am imagining `gratipay.coop` in the address bar.

---------------------

![screen shot 2017-01-12 at 11 26 40 am](https://cloud.githubusercontent.com/assets/134455/21898314/7b521470-d8ba-11e6-8bd2-3551ad36ace0.png)

---------------------

![screen shot 2017-01-12 at 11 26 57 am](https://cloud.githubusercontent.com/assets/134455/21898315/7b53dfda-d8ba-11e6-9b65-7e691d42c2d3.png)

---------------------

![screen shot 2017-01-12 at 11 27 05 am](https://cloud.githubusercontent.com/assets/134455/21898313/7b4f4e2a-d8ba-11e6-99dd-f3d656979bc5.png)

**Comment by whit537** [comment]
From @jdaviescoates in https://open.coop/2016/06/23/not-a-co-op-not-a-platform-co-op/:

> I **much** prefer this [perspective](https://gratipay.news/a-platform-cooperative-almost-94cd5900840f#.iv4x8jq19) from the good people at [Gratipay](https://gratipay.com/); they are very much moving in the right direction and are very closely aligned to both the Platform Co-op and Open Co-op visions, but are in full awareness that they aren’t quite there yet.

☺️

> Regardless of legal structure (there are actually plenty of unincorporated co-ops), the key thing is that to be considered a co-op (and therefore a Platform Co-op) you ought to uphold the [internationally agreed co-operative values and principles](https://open.coop/co-operative/).

That would be easy enough for us to do by inlining them into our [values statement](http://inside.gratipay.com/big-picture/brand), and would even seem to be enough to make us [eligible](http://www.nic.coop/eligibility/) for a `.coop` domain ("Are you [...] an organization that is committed to the seven cooperative principles [...] Then you are eligible for a .coop domain!"). Hmm ... :thinking:

>Ownership really does matter. A lot. And allowing “Platform Co-op” to mean “a company started by nice people with good intentions who also built an app” will result in the term quickly becoming meaningless.

I still think there's something to be said for the open source principle of forkability as an accountability measure. There's a sense in which forkability is the ultimate democratic ownership, as @Changaco has, I think, amply demonstrated. :grin:

**Comment by nobodxbodon** [comment]
I wonder what it means if becoming a co-op:
Internally, what procedures will be different from now? And externally, what changes will be made to the user terms?

**Comment by nobodxbodon** [comment]
Besides, I sincerely hope our limited resources can be focused on *the* [project](https://github.com/gratipay/inside.gratipay.com/pull/981), especially if we put it in roadmap publically, unless a structural change is absolutely necessary before that project is launched.

**Comment by Changaco** [comment]
> There's a sense in which forkability is the ultimate democratic ownership, as @Changaco has, I think, amply demonstrated. 😁

I don't think that forkability can be likened to ownership, it's more a lack of monopoly on the code. While forkability allowed me to create Liberapay "quickly" (it still took me 8 months), democratic ownership would have allowed every Gratipay member to vote with equal power on the choices that had to be made, which may or may not have resulted in different decisions than the ones that were taken.

Liberapay isn't a co-op, but it has co-directors (currently 4 of them) who each have one vote on every decision taken by its nonprofit organization (that includes any use of donations received by the org, changes to the terms of service, etc).

**Comment by whit537** [comment]
> democratic ownership would have allowed every Gratipay member to vote with equal power on the choices that had to be made

That depends on the details of the democracy. Which decisions are constitutional and which are operational? Let's say everything went down exactly as it did, with the exception that yinz had voted me in as CEO or president or director or whatever. That would've counted for a democratic workplace under cooperative principles, even though we didn't hold a formal vote on the particular decision to start banning people (a decision I discussed at length with a number of people, particularly at the [retreat that year](https://gratipay.news/at-our-second-retreat-we-met-our-third-users-5b473a2d1464) as well as with you and others on https://github.com/gratipay/violations/issues/1).

What's more, you mentioned in person that you probably would've forked even if we _had_ voted and the vote led us in the same direction we in fact took. The bottom line is that the situation we found ourselves in was a conflict-ridden mess, and it's not clear that a fine-grained democratic constitution such as Liberapay's would've performed any better (and for what definition of "performed"?) given the circumstances. Gratipay's forkability, on the other hand, at least allowed you to retain some measure of autonomy.

It was interesting to discover the following in [*Ours to Hack and to Own*](http://www.orbooks.com/catalog/ours-to-hack-and-to-own/) a couple days ago, in [Mayo Fuster Morell](https://en.wikipedia.org/wiki/Mayo_Fuster_Morell)'s description of platform provision as one of "six interrelated factors as determinants and drivers of commons governance" (214):

> For participants, autonomy is linked to the license held for the commons-pool resources and the type of software used for the platform (i.e., copyleft licenses and the use of freely available code, as in FLOSS, versus conditions defined by ordinary copyright). If the platform can be replicated—if it is "forkable"—the relationships created on forked versions are free from the original platform provider. FLOSS and copyleft licensing allow platforms to be replicated, while close copyright license regimes prohibit that. In other words, the use of FLOSS and a copyleft license creates conditions in which the community can have greater autonomy and freedom from the platform provider.

Though she immediately goes on to say:

> Governance very much depends on who is in control of these six power nodes in commons production. Each of the axes of governance can be managed in an inclusive or exclusive way. They may encourage involvement on the basis of participants as individuals or through the community as a whole.

([Miriam Cherry](http://law.slu.edu/people/miriam-cherry)'s chapter, the next one, is even more highly relevant to our situation.)

**Comment by whit537** [comment]
> unless a structural change is absolutely necessary before that project is launched.

It's not _absolutely_ necessary, of course. But I think a more formal identification with the cooperative movement could solidify our identity for ourselves, and also help to distinguish us from (in particular, at the moment) OpenCollective, which is a VC-backed for-profit.

I think the proposal I'm arriving at here is that we:

- adopt the cooperative principles into our values
- apply for a `.coop` domain with no further changes
- let the cooperative identity guide the future evolution of our constitutional structure

In other words, we may be closer than we think to being able to close this ticket! 😱

**Comment by Changaco** [comment]
> The bottom line is that the situation we found ourselves in was a conflict-ridden mess, and it's not clear that a fine-grained democratic constitution such as Liberapay's would've performed any better (and for what definition of "performed"?) given the circumstances.

Of course we'll never know for sure whether formal rules similar to Liberapay's would have produced a better outcome or not, but I think they might have helped us reach a compromise.

> Gratipay's forkability, on the other hand, at least allowed you to retain some measure of autonomy.

I'm not denying that, I merely said that forkability isn't "the ultimate democratic ownership".

**Comment by whit537** [comment]
> we'll never know

Indeed! Such tantalizing what-ifs! :-)

> I merely said that forkability isn't "the ultimate democratic ownership".

Sure, not in the sense of "ultimate" as best or most thorough or highly developed or something like that. I was thinking of "forkability as an accountability measure." Forkability provides a worst-case scenario that is more democratic (preserving of individual autonomy; non-monopolistic) than the worst-case scenario in a situation lacking forkability. The proper constitution for the best of times is another matter.

**Comment by whit537** [comment]
Forkability raises the lower bound on democracy, in other words.

**Comment by whit537** [comment]
> ([Miriam Cherry](http://law.slu.edu/people/miriam-cherry)'s chapter, the next one, is even more highly relevant to our situation.)

Here is her conclusion (pp. 226-227):

> Regardless of the choice of business entity, another important issue is designing a workable governance structure in the operating agreement or corporate documents. [&hellip;]
>
> The fact that there are no tailor-made enabling statutes geared specifically toward platform cooperatives contributes to increased setup costs and barriers to entry. But many businesses that do not fit the traditional mold have had to confront this issue before. [&hellip;]
>
> Given the turnover and flexibility of online platform work, the operating documents should be written to allow for relative ease of entry and exit as a member. In addition, the organizing documents must also set up the relationship in a way that sets out what the expectations are for the members, clearly and succinctly. The documents need to include provisions for reducing the share of profits if an individual member is shirking, and also contain clear provisions defining under what circumstances a member or shareholder may be disassociated. In terms of the global or international scope of many platforms, the operating agreement and other documents can be written to provide for choice of law and choice of jurisdiction. Current statutes allow for electronic or remote voting for boards of directors or members, so long as such procedures are set out in the corporate charter or operating agreement. Note that running a  business is riskier for the individual worker in a platform cooperative—like any business, the LLC members or B Corporation shareholders run the risk that there will be no profits.
>
> Perhaps the answer to the misclassification lawsuits and the struggle over employee status is to work around it, regardless of the outcome. While not the perfect solution, already-existing legal structures can be modified to accommodate platform cooperatives.

**Comment by whit537** [comment]
I've kicked off [a project](https://github.com/orgs/gratipay/projects/17) to encompass this ticket and related work.

**Comment by whit537** [comment]
Sticking with this ticket for the time being for the legal/structural discussion ... what if we formalize three categories of legal relation to Gratipay, LLC:

- volunteer,
- contractor, and
- owner.

"Collaborator" covers all three classes, as all are members of the Gratipay project on Gratipay (pending https://github.com/gratipay/gratipay.com/issues/4299), and all compensation runs through twyw and looks the same from that point of view. Perhaps the requirements to become an owner are something like:

1. At least two years as a non-owner collaborator (i.e., a volunteer and/or contractor).
1. Permission from the United States to co-own a US LLC (basically, an SSN or ITIN).
1. Consensus of the existing owners.

**Comment by whit537** [comment]
Re: non-cooperative operating agreements:

https://en.wikipedia.org/wiki/Operating_agreement
http://www.northwestregisteredagent.com/pdf/operating-agreement.pdf

**Comment by whit537** [comment]
[Draft bylaws](https://docs.google.com/document/d/1SYuw2Nhc7yhEaR2Q0gd0goDAojKoIx85d1JqlZypnCY/edit#heading=h.ejuqez34g3bk) (!m @clone1018.). How do bylaws relate to operating agreement?

**Comment by whit537** [comment]
bylaws:corporation::operating agreement:LLC

https://www.google.com/search?q=bylaws+operating+agreement

So we need an operating agreement (and not bylaws).

**Comment by whit537** [comment]
Many tabs open, down the rabbit hole.

[CHAPTER 88: LIMITED LIABILITY COMPANIES](http://www.legis.state.pa.us/cfdocs/legis/LI/consCheck.cfm?txtType=HTM&ttl=15&div=0&chpt=88)

![screen shot 2017-05-15 at 4 45 51 pm](https://cloud.githubusercontent.com/assets/134455/26078567/fc3ca120-398d-11e7-8a5f-0c4607a14622.png)

**Comment by whit537** [comment]
## LLCs in General

LegalZoom [overview of operating agreements](https://www.legalzoom.com/articles/how-to-draft-an-llc-operating-agreement) (non-cooperative)

[A random member-managed operating agreement](https://www.delawareinc.com/forms/member_managed_llc_operating_agreement_template.pdf) (non-cooperative)


## Worker-owned Co-ops in General

More [general guidance on coop structure](http://www.co-oplaw.org/governance-operations/entity/) from SELC.

[Choosing a Business Entity](http://www.co-oplaw.org/wp-content/uploads/2015/02/ChoicofEntityFinal.pdf)


## Worker-owned Co-ops as LLCs

[general guidance on worker-owned coops as LLC](http://www.e4fc.org/images/limited-liability-co-worker-coop.pdf)

@JanelleOrsi's [cartoon bylaws](https://drive.google.com/file/d/0B7aFfmk4wQfbVkVYSFNVWTZyZUU/view) for a particular worker-owned coop

## Details, details, details ...

Tax implications for members:
 - [LLC is more complicated than a C-Corp](http://cdi.coop/coop-llc-individual-member-taxes/)
 - LLC is supposedly easier to deal in foreign owners—a definite consideration for us—but the tax hit is then 35-40%(!!) ... and [are C-Corps really that hard](http://www.cohencpa.com/insights/articles/llc-vs-c-corp-entity-structures-for-foreign-owne)?

[active vs. passive members](https://www.entrepreneur.com/article/57370) ([etc.](http://www.thetaxadviser.com/issues/2013/feb/casestudy-feb2013.html), [etc.](http://www.thetaxadviser.com/issues/2010/sep/nash-sept2010.html), [etc.](http://www.thetaxadviser.com/issues/2013/feb/casestudy-feb2013.html), [etc.](https://www.bogleheads.org/forum/viewtopic.php?t=111981))

[splitting profits](http://info.legalzoom.com/profits-split-llc-20063.html) and [equity incentives](https://www.nceo.org/articles/equity-incentives-limited-liability-company-llc)

[loan money to one's own LLC](http://info.legalzoom.com/can-lend-money-own-llc-business-23184.html)

manager-managed vs. member-managed

taxed as partnership vs. corporation

[guaranteed payments vs. distributions](https://www.entrepreneur.com/article/30062) (old but seemingly [still applicable](https://www.forbes.com/sites/anthonynitti/2016/10/03/irs-grows-increasingly-aggressive-in-subjecting-income-of-llc-member-to-self-employment-tax/#6ba12d078a81))

([Indie.vc wouldn't need to be a member](https://github.com/indievc/indievc/blob/master/Term%20Sheet%20v.2)

**Comment by whit537** [comment]
To: CPA

> If I bring on a foreign partner in Gratipay, LLC, I believe we'll need to [withhold 39.6%](https://www.irs.gov/individuals/international-taxpayers/partnership-withholding) of payments to them (whether guaranteed payments or distributions proper), yes? But then it also sounds like they can file for a refund, yes? How do we estimate how much their refund will be and therefore what net tax burden to expect?

**Comment by whit537** [comment]
> Yes,  you need to withhold on those payments and amount to withhold and remit to the IRS is 35%. A good estimate on the net tax amount is about 25%

**Comment by whit537** [comment]
The tax examples in http://cdi.coop/coop-llc-individual-member-taxes/ come out to 30-35% for a domestic member. Close enough to that of a foreign member that I don't think we should worry about any discrepancy there. The ease with which we can bring on foreign members to an LLC vs. C-corp more than makes up for a percentage point or three.

**Comment by whit537** [comment]
The "minimalist six-member board" idea above at https://github.com/gratipay/inside.gratipay.com/issues/72#issuecomment-166668883 is bad because we want people who hang around here a long time to become owners rather than employees.

**Comment by whit537** [comment]
Starting over with the https://github.com/gratipay/inside.gratipay.com/issues/72#issuecomment-301618663 stew in my head ...

**Comment by whit537** [comment]
Moving to a PR in https://github.com/gratipay/inside.gratipay.com/pull/1074.

Draft is taking shape here:

https://gratipay-operating-agreement.herokuapp.com/big-picture/operating-agreement

**Comment by whit537** [comment]
Gratipay now has [a cooperative operating agreement](http://inside.gratipay.com/big-picture/operating-agreement)! 📜 💃 🌻 

A hearty **thank you** to everyone who participated in this thread! 👏 👍 💯
